src/logger.ts

```diff
@@ -0,0 +1,70 @@
+import * as winston from 'winston';
+import * as os from 'os';
+// import * as Elasticsearch from 'winston-elasticsearch';
+const Elasticsearch = require('winston-elasticsearch');
```

Comment:
why not use `import`?
also name it `winstonElasticsearch` and not `Elasticsearch`, if you have to keep it with `require` then put it after the `import` lines.

Reply:
done, the `require` left was agreed upon with @omrishtam